extensions/replication/utils/ReplicationTaskScheduler.js

```diff
@@ -1,24 +1,57 @@
 const async = require('async');
 
+const ObjectQueueEntry = require('../../../lib/models/ObjectQueueEntry');
+const ActionQueueEntry = require('../../../lib/models/ActionQueueEntry');
+
 /**
- * Schedule replication tasks for replicated buckets, by ensuring that
- * all updates to the same object are serialized
-*/
+ * Schedule replication tasks by ensuring that all updates to the same object
```

Comment:
Maybe you can also explain a bit about the just-introduced mechanism to skip entries :)